db/migrate/20221117233130_no_upload_type_default.rb

```diff
@@ -0,0 +1,11 @@
+class NoUploadTypeDefault < ActiveRecord::Migration[7.0]
+  def up
+    change_column_default(:work_versions, :upload_type, nil)
+    change_column_null(:work_versions, :upload_type, true)
+  end
+
+  def down
+    change_column_default(:work_versions, :upload_type, 'browser')
+    change_column_null(:work_versions, :upload_type, false)
+  end
+end
```

Comment:
we actually need to allow the column to be nil and have no default...since otherwise PURL reservations will not work correctly (unless we default them to "browser" upload type as was happening until now)

we will force the user to make a choice by a validation on the work form